### ISSUE TITLE
Pre-rooting update .up file

### ISSUE THREAD

**Issue by przemhb** [Trevelopment/MZD-AIO#201]
Suppose we knew password for the .up file, which is in fact encrypted ZIP archive. Correct me if I am wrong, but we could just modify appropriate files (autorun?), compress it back again with the same pass and flash CMU with it.

**Comment by ktkd** [comment]
Nope, updater checks a signature of update file and fails if you are modify it.
